test/parallel/test-http2-session-timeout.js

```diff
@@ -6,21 +6,23 @@ if (!common.hasCrypto)
   common.skip('missing crypto');
 const h2 = require('http2');
 
-const serverTimeout = common.platformTimeout(200);
-const callTimeout = common.platformTimeout(10);
+const serverTimeout = common.platformTimeout(2000);
+const callTimeout = common.platformTimeout(300);
```

Comment:
For me, increasing `callTimeout` increases the test flakiness rather than decreasing it. 

Leaving all your other changes in place, if I set it to `1`, the test doesn't fail under a local stress test. If I set it to `1900`, it fails.

Contrary information welcome, of course.

Reply:
Argh, no I think I'm wrong. Ignore!